src/ring_theory/ring_hom/surjective.lean

```diff
@@ -0,0 +1,78 @@
+/-
+Copyright (c) 2022 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import ring_theory.local_properties
+
+/-!
+
+# The meta properties of surjective ring homomorphisms.
```

Comment:
I'd prefer if we avoid the use of `meta` in this sense: in mathlib it's only going to be confused with the keyword `meta` (i.e. referring to the machinery to manipulate proofs and the environment).

Reply:
@semorrison I'm not sure if I have a good alternative to it, and other files in this folder also uses such a title ([example](https://leanprover-community.github.io/mathlib_docs/ring_theory/ring_hom/finite.html)).

I could replace "meta property" with "property", but I think this would cause some confusion as well. If you think this is still better than the current title, I'm happy to change it.

Reply:
Okay. I guess we are deep enough into mathematics by now that people can start use "meta" without meaning `meta`. :-)

bors merge